packages/v-connection-string/test/parse.js

```diff
@@ -194,29 +194,25 @@ describe('parse', function () {
     subject.options.should.equal('-c geqo=off')
   })
 
-  it('configuration parameter ssl=true', function () {
-    var connectionString = 'pg:///?ssl=true'
+  it('configuration parameter tls_mode=require', function () {
+    var connectionString = 'pg:///?tls_mode=require'
     var subject = parse(connectionString)
-    subject.ssl.should.equal(true)
+    subject.tls_mode.should.equal('require')
   })
 
-  it('configuration parameter ssl=1', function () {
-    var connectionString = 'pg:///?ssl=1'
+  it('configuration parameter tls_mode=disable', function () {
+    var connectionString = 'pg:///tls_mode=disable' 
     var subject = parse(connectionString)
-    subject.ssl.should.equal(true)
+    subject.tls_mode.should.equal('disable')
   })
 
-  it('configuration parameter ssl=0', function () {
-    var connectionString = 'pg:///?ssl=0'
-    var subject = parse(connectionString)
-    subject.ssl.should.equal(false)
-  })
-
-  it('set ssl', function () {
-    var subject = parse('pg://myhost/db?ssl=1')
-    subject.ssl.should.equal(true)
+  it('set tls_mode', function () {
+    var subject = parse('pg://myhost/db?tls_mode=require')
+    subject.tls_mode.should.equal('require')
   })
 
+  // MUTUAL MODE TESTS - ENABLE WITH mTLS
+  /*
   it('configuration parameter sslcert=/path/to/cert', function () {
```

Comment:
I still think we should use `.skip()` on `describe` or `it` blocks we want to disable instead of commenting them out as it will be easier to track/find disabled tests (skipped tests will show up in the test report but will not cause it to fail).  Although I suppose there will need to be a ticket to go back and find all the commented out tests and fix them.

Reply:
You are right that would probably be the best practice moving forward. This was just a quick way to get this out of the way while we're pressed for time. It's already logged in an mTLS issue that's assigned to me to get these tests updated/working again so it won't be forgotten about. Also the describe/it blocks you are also right about. It seems we have packages/tests that don't have the same testing framework, or have it and don't use it. There is some broader testing overhaul work that should be done, but the priority of that is probably not very high compared to the other driver work we have

Reply:
Sounds good.   I'll approve the PR.   Since Danny is in Montana off the grid, I'll go ahead and merge it.